lib/Redmine/Api/AbstractApi.php

```diff
@@ -117,6 +117,36 @@ protected function sanitizeParams(array $defaults, array $params)
             array($this, 'isNotNull')
         );
     }
+
+    /**
+     * Generate a Redmine compatible URL-encoded query string
+     * 
+     * @param mixed $data
+     * @param string $parentKey
+     * @param bool $isNested
+     * @return string string a URL-encoded string.
+     */
+    function httpBuildQuery($data, $parentKey = '', $isNested = false)
```

Comment:
Thanks for this proposal, although i'm not very keen at re-inventing the wheel! 
How about trying to add sort of a "potHttpBuildQuery" processor instead?
see this comment: http://php.net/manual/en/function.http-build-query.php#111819

If you can confirm that this 1 liner fixed your issue I would go in favor of that!